corehq/apps/es/management/commands/restore_es_snapshot.py

```diff
@@ -88,6 +88,6 @@ def rewind_pillows(date):
                                                                     date_updated=date)
                 seq = checkpoint.seq
             except HistoricalPillowCheckpoint.DoesNotExist:
-                seq = DEFAULT_EMPTY_CHECKPOINT_SEQUENCE
+                seq = DEFAULT_EMPTY_CHECKPOINT_SEQUENCE[checkpoint.sequence_format]
```

Comment:
since this is in the exception handler for the case where the `HistoricalPillowCheckpoint` doesn't exists `checkpoint` should still point to `pillow.checkpoint`. But I agree that it's not clear. I'll update.